monkey/monkey_island/cc/setup/island_config_options.py

```diff
@@ -14,8 +16,16 @@ def __init__(self, config_contents: dict):
         self.data_dir = os.path.expandvars(
             os.path.expanduser(config_contents.get("data_dir", DEFAULT_DATA_DIR))
         )
+
         self.log_level = config_contents.get("log_level", DEFAULT_LOG_LEVEL)
 
         self.start_mongodb = config_contents.get(
             "mongodb", {"start_mongodb": DEFAULT_START_MONGO_DB}
         ).get("start_mongodb", DEFAULT_START_MONGO_DB)
+
+        self.crt_path = os.path.expandvars(
+            os.path.expanduser(config_contents.get("cert_path", DEFAULT_CRT_PATH))
+        )
+        self.key_path = os.path.expandvars(
+            os.path.expanduser(config_contents.get("key_path", DEFAULT_KEY_PATH))
+        )
```

Comment:
We're making multiple calls to `expandvars()` and `expanduser()`. Can we add a utility function in this file that wraps this functionality?